website.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: deploy pyslackers website
+  hosts: all
+  vars:
+    domain: pyslackers.com
+    python_version: 3.6.1
+    repo: https://github.com/pyslackers/website.git
+    service_name: pythondev-site
+    site_port: 3000
+    ssl: false
+    username: pythondev
+    version: master
+
+  roles:
+    - common
+    - user
+    - nginx
+    - pythonapp
+
+    - role: system_service
```

Comment:
I don't love this API around the role, but haven't thought of a better way that remains generic enough...